src/GraphNodeManagerViewExtension/ViewModels/GridNodeViewModel.cs

```diff
@@ -155,6 +159,35 @@ internal set
                 RaisePropertyChanged(nameof(StateIsFunction));
             }
         }
+        public bool IsOutputSingleItem
+        {
+            get
+            {
+                outputIsSingleItem = IsNodeOutputSingeItem(NodeModel.CachedValue);
+                return outputIsSingleItem;
```

Comment:
For the getter, we usually try to make it concise, if no temp variable needed, let's just `return IsNodeOutputSingeItem(NodeModel.CachedValue);`